tox.ini

```diff
@@ -0,0 +1,66 @@
+[tox]
+min_version = 4.14.0
+envlist =
+    lint
+    tests
+
+
+[testenv]
+package = external
+package_env = build-chemiscope
+lint_folders = "{toxinidir}/python" "{toxinidir}/setup.py"
+
+
+[testenv:build-chemiscope]
```

Comment:
I think simply `build` is enough. We are not building different flavors as the the other bigger packages.

```suggestion
[testenv:build]
description = builds the package and checks integrity
```

Reply:
done!